index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>SergSkol Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  </head>
+  <body>
+    <header class="header">
+        <a class="logo" href="#">
+          Serg Skol
+        </a>
+        <nav class="menu">
+            <a class="menu_icon fa-solid fa-bars" title="Menu"></a>
+        </nav>
+    </header>
+    <section class="headline">
+      <br><br><br>
+      <h1 class="headline_top">I'm Sergiy<br>Glad to see you!</h1>
+      <p class="headline_text">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <a class="headline_connect">LET'S CONNECT</a>
+      <ul class="nobull links">
+        <li>
+          <a class="social fa-brands fa-twitter" href="http://twitter.com/sps676"></a>
+        </li>
+        <li>
+          <a class="social fa-brands fa-linkedin" href="https://www.linkedin.com/in/sergiy-shkolnik-7801a53/"></a>
+        </li>
+        <li>
+          <a class="social fa-brands fa-github" href="https://github.com/SergSkol"></a>
+        </li>
+      </ul>
+    </section>
```

Comment:
Hi @SergSkol, Linters only flagged errors because the letters in the colors are in capital uppercase. Kindly change the letters to lower case and your work should run without linters errors.

Reply:
Thank you! that's my mistake, let me fix it...